images/ceph/Makefile

```
@@ -121,3 +124,15 @@ csv: $(OPERATOR_SDK) $(YQ) ## Generate a CSV file for OLM.
 
 csv-clean: $(OPERATOR_SDK) $(YQ) ## Remove existing OLM files.
 	@rm -fr ../../cluster/olm/ceph/deploy/* ../../cluster/olm/ceph/templates/*
+
+# list-image creates list of images for offline installation
+list-image:
+	@echo "producing list of images for offline installation";\
+	# remove the file if already exists
+	[ -f $(MANIFESTS_DIR)/images.txt ] && rm $(MANIFESTS_DIR)/images.txt;\
```

```suggestion
	rm -f $(MANIFESTS_DIR)/images.txt;\
```

`rm -f` is idempotent, you don't need to check the existence of the file.